src/baitapquanlyphuongtiengiaothong/QuanLyPTGT.java

```diff
@@ -0,0 +1,108 @@
+package baitapquanlyphuongtiengiaothong;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Scanner;
+
+public class QuanLyPTGT {
+	int soLuongOto = 0;
+	int soLuongXeMay = 0;
+	int soLuongXeTai = 0;
+	private ArrayList<PhuongTienGiaoThong> ds = new ArrayList<PhuongTienGiaoThong>();
+
+	public void nhapPhuongTien(Scanner sc) {
+		System.out.print("Nhập loại phương tiện giao thông: ");
+		String loaiXe = sc.nextLine();
+		switch (loaiXe) {
+		case "OTo":
+			OTo oTo = new OTo();
+			oTo.nhap(sc);
+			ds.add(oTo);
+			soLuongOto++;
+			break;
+		case "XeMay":
+			XeMay xeMay = new XeMay();
+			xeMay.nhap(sc);
+			ds.add(xeMay);
+			soLuongXeMay++;
+			break;
+		case "XeTai":
+			XeTai xeTai = new XeTai();
+			xeTai.nhap(sc);
+			ds.add(xeTai);
+			soLuongXeTai++;
+			break;
+		default:
+			System.out.println("Không có loại xe bạn cần nhập!");
+			System.out.println("Vui lòng nhập lại: ");
+			nhapPhuongTien(sc);
+		}
+	}
+
+	public void themnhieuPhuongtien(Scanner sc) {
+		while (true) {
+			nhapPhuongTien(sc);
+			System.out.println("=======================================");
+			System.out.print("Bạn có muốn thêm phương tiện giao thông không? (y/n): ");
+			String xacNhan = sc.nextLine();
+			if (xacNhan.equals("n")) {
+				break;
+			}
+		}
+	}
+
+	public void timPhuongTienTheoSoXe(Scanner sc) {
+		System.out.print("Nhập số xe muốn tìm kiếm: ");
+		int soXe = Integer.parseInt(sc.nextLine());
+		for (int i = 0; i < ds.size(); i++) {
+			PhuongTienGiaoThong phuongTien = ds.get(i);
+			if (phuongTien.getSoXe() == soXe) {
+				phuongTien.hienThongTin();
+			}
+		}
+	}
+
+	public void timPhuongTienTheoCMND(Scanner sc) {
+		System.out.print("Nhập số CMND muốn tìm kiếm: ");
+		int cmnd = Integer.parseInt(sc.nextLine());
+		for (int i = 0; i < ds.size(); i++) {
+			PhuongTienGiaoThong phuongTien = ds.get(i);
+			if (phuongTien.getChuXe().getSoCMND() == cmnd) {
+				phuongTien.hienThongTin();
+			}
```

Comment:
ko cần phải gán đối tượng như line 70 đâu em nhé
```suggestion
			if (phuongTien.getChuXe().getSoCMND() == cmnd) {
				ds.get(i).hienThongTin();
			}
```